markov/markov.py

```diff
@@ -0,0 +1,476 @@
+import logging
+import random
+import re
+from io import BytesIO
+
+import discord
+from redbot.core import Config, checks, commands
+from redbot.core.utils.mod import is_mod_or_superior
+
+log = logging.getLogger("red.rhomelab.markov")
+
+__all__ = ["UNIQUE_ID", "Markov"]
+
+UNIQUE_ID = 0x6D61726B6F76
```

Comment:
Probably want to change this ID as it's a fork of an existing cog with (presumably) the same name